### PR TITLE
Create stale_marker.yml

### PR DESCRIPTION
**Pull request by morkovka1337**
Add automatic stale mark for old issues and PRs.

### CODE REVIEW

**Comment by nervana-ff** [comment]
Can one of the admins verify this patch?

File: .github/workflows/stale_marker.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
```
Comment (Ilya-Krylov) [inline .github/workflows/stale_marker.yml]: too short period of time

Comment (morkovka1337) [reply .github/workflows/stale_marker.yml]: I usually look through the issues and manually close those ones that did not have activity during past 1.5-2 months. What is your suggestion about period? 3 months?

Comment (Ilya-Krylov) [reply .github/workflows/stale_marker.yml]: yeah, let it be 3 month

File: .github/workflows/stale_marker.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+          stale-pr-message: 'This PR is stale because it has been open 60 days with no activity.'
+          close-issue-message: 'This issue was closed because it has been stalled for 7 days with no activity.'
+          days-before-issue-stale: 45
+          days-before-pr-stale: 60
+          days-before-issue-close: 7
+          days-before-pr-close: -1
```
Comment (Ilya-Krylov) [inline .github/workflows/stale_marker.yml:18]: what does it mean?

File: .github/workflows/stale_marker.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
+          stale-pr-message: 'This PR is stale because it has been open 90 days with no activity.'
+          close-issue-message: 'This issue was closed because it has been stalled for 7 days with no activity.'
+          days-before-issue-stale: 90
+          days-before-pr-stale: 90
+          days-before-issue-close: 7
```
Comment (Ilya-Krylov) [inline .github/workflows/stale_marker.yml]: 30

File: .github/workflows/stale_marker.yml (hunk 2 above)
Comment (morkovka1337) [reply .github/workflows/stale_marker.yml:18]: Stale PRs would not be closed

File: .github/workflows/stale_marker.yml (hunk 3 above)
Comment (morkovka1337) [reply .github/workflows/stale_marker.yml]: Updated

File: .github/workflows/stale_marker.yml (hunk 2 above)
Comment (Ilya-Krylov) [reply .github/workflows/stale_marker.yml:18]: what is it for then?

Comment (morkovka1337) [reply .github/workflows/stale_marker.yml:18]: Mark it as stale. By default, all stale issues and PRs are closed.

Comment (Ilya-Krylov) [reply .github/workflows/stale_marker.yml:18]: yours statements contradict each other.

Comment (morkovka1337) [reply .github/workflows/stale_marker.yml:18]: I mean, if not specify explicitly `days-before-pr-close: -1`, PR will be closed after default timeout period.

**Pull request merged**